parts/windows/windowscsehelper.ps1

```diff
@@ -217,3 +217,12 @@ function Assert-FileExists {
         Set-ExitCode -ExitCode $global:WINDOWS_CSE_ERROR_FILE_NOT_EXIST -ErrorMessage "$Filename does not exist"
     }
 }
+
+function Get-WindowsVersion {
+    $buildNumber = (Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").CurrentBuild
+    switch ($buildNumber) {
+        "17763" { return "1809" }
+        "20348" { return "ltsc2022" }
+        Default { throw "It doesn't supports the windows build number: $buildNumber" }

```

Comment:
Please define a new error in parts/windows/windowscsehelper.ps1 and then throw it here. 
For example,
$global:WINDOWS_CSE_ERROR_NOT_FOUND_BUILD_NUMBER=29